### PR TITLE
Add the SLP7 Release Note and Mention About SLP6 in Bio

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the SLP7 to the release note and mention SLP6 in Bio.

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-preview7/RELEASE_NOTE.md [hunk 1]
```diff
@@ -0,0 +1,369 @@
+---
+layout: ballerina-blank-page
+title: Release Note
+---
+### Overview of Ballerina Swan Lake Preview 7 
+
+This release is the sixth preview version of Ballerina Swan Lake. This release includes a new set of language features along with improvements and bug fixes to the compiler, runtime, standard library, and developer tooling.
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-preview7/RELEASE_NOTE.md:7]:
```suggestion
This release is the seventh preview version of Ballerina Swan Lake. This release includes a new set of language features along with improvements and bug fixes to the compiler, runtime, standard library, and developer tooling.
```

**Comment by anupama-pathirage** [comment]
@praneesha IMO, we need to add this internal version message to https://ballerina.io/downloads/swan-lake-release-notes/ page as well.

**Comment by praneesha** [comment]
> @praneesha IMO, we need to add this internal version message to https://ballerina.io/downloads/swan-lake-release-notes/ page as well.

This addresses it: https://github.com/ballerina-platform/ballerina-dev-website/blob/c85c81e40eff02da8e17194120f1f9b6d52b8c11/downloads/swan-lake-release-notes/swan-lake-preview6/RELEASE_NOTE.md

**Review by anupama-pathirage — APPROVED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]

File: downloads/swan-lake-archived.md [hunk 2]
```diff
@@ -19,10 +19,22 @@ permalink: /downloads/swan-lake-archived/
         <div class="col-xs-12 col-sm-16 col-md-12 col-lg-12">
             <div class="cStandaloneInstallers" id="archived-versions">
                 <h2>Swan Lake Archived Versions</h2>
+                <div class="cInstallers" style="height:150px"> 
+                <h3 class="release-version">swan-lake-preview6</h3>
+<div class="releases-row">
+    <div class="col-xs-12 col-sm-16 col-md-6 col-lg-6 cLeftTable">
+
+This release is an internal release only.
```
Comment (MaryamZi) [inline downloads/swan-lake-archived.md:27]: @praneesha, should this be "This release is an internal-only release." or "This release is internal only." instead?

Comment (praneesha) [reply downloads/swan-lake-archived.md:27]: This is done via https://github.com/ballerina-platform/ballerina-dev-website/pull/1565/files.

**Pull request merged**